Qumran.sln.DotSettings

```diff
@@ -0,0 +1,3 @@
+﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
```

Comment:
Sorry, I don't know what this is.  I will look into it, but it must be something my IDE added.

Reply:
I guess this is ReSharper settings that can be shared with others working on the project.  I guess I will leave it, since it may be useful to Ingo too.